src/realtime_metrics/main.py

```diff
@@ -395,6 +412,60 @@ def get_next_actual_arrival(timestamp: int, route_id: str, stop_id: str) -> Stop
     return actual_arrivals[0]
 
 
+def prediction_inconsistency(actual_arrival_time: int, updates: list[tuple[TripUpdate, StopTimeUpdate]]) -> float:
+    """
+    Computes the prediction inconsistency metrics for a route, trip and stop combination.
```

Comment:
A route is redundant, because every trip and stop combination only has one route (due to the trip only having one route)
```suggestion
    Computes the prediction inconsistency metrics for a trip and stop combination.
```

Reply:
As discussed, as long as we also use `route_id` as part of the identifier, we leave this is. In the future, we can check if we can remove it as part of the identifier and then also remove it from the documentation.